### PR TITLE
Update banner design

### PR DESCRIPTION
**Pull request by Nikomus**
- Update donation banner design

https://neontribe.atlassian.net/browse/SEAP-134

### CODE REVIEW

File: src/assessment.handlebars [hunk 1]
```diff
@@ -1046,8 +1046,9 @@
   </div>
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
-      <div class="center donation-banner">Text SEAP02 £2 to 70070 to help us support more people like you.
-          <img src="images/jtg_logo.png" alt="Just Text Giving logo"/>
+      <div class="donation-banner">
+        <p class="center">If you find this website useful, please consider donating.</p>
+        <p class="center">Text <b>'SEAP02 £2'</b> to <b>70070</b> to help us support more people like you.</p>
```
Comment (katjam) [inline src/assessment.handlebars:1051]: I think we should use `&#x2018;` and `&#x2019;` to be consistent with the existing encoding in these apps.

Comment (katjam) [reply src/assessment.handlebars:1051]: It doesn't seem massively consistent use of `&#39;` for single quote too - which should be apostrophe.

But we might as well use the correct open and close single encode for this update.

**Review by katjam — CHANGES_REQUESTED** [review]
Use html entity for single quotes

**Comment by katjam** [comment]
Looking at it now - there are a few places where `'` has been used as well... so your call. Probably not wast time - we can normalise when/ if we migrate.

**Review by katjam — APPROVED** [review]
Fine as it is

**Comment by Nikomus** [comment]
As this is the approved design and there is some inconsistency already with quote/apostrophe usage we are merging as is with a view to review again in a later release.

**Pull request merged**